src/components/molecules/search/__tests__/Search.test.js

```diff
@@ -230,7 +230,8 @@ describe('<Search /> English', () => {
 				fireEvent.click(searchButton);
 			});
 			const expectedLocationObject = {
-				pathname: `/search/meta-analysis/`,
```

Comment:
did this linkage change?

Reply:
Whoops I changed it a while ago to test something and forgot to switch it back. Fixed now.

Reply:
Ah wait getting an issue on one of the unit tests that I missed.

Reply:
Ok should be fine now